src/librustdoc/passes.rs

```diff
@@ -242,6 +239,21 @@ impl<'a> fold::DocFolder for ImplStripper<'a> {
     }
 }
 
+// This stripper discards all private import statements (`use`, `extern crate`)
+struct ImportStripper;
+impl fold::DocFolder for ImportStripper {
+    fn fold_item(&mut self, i: Item) -> Option<Item> {
+        match i.inner {
+            clean::ExternCrateItem(..) |
```

Comment:
This should also test visibility, right? https://github.com/rust-lang/rust/pull/31362 made `pub extern crate` meaningful.